quill-core/src/main/scala/io/getquill/dsl/MetaDslMacro.scala

```diff
@@ -127,17 +125,17 @@ class MetaDslMacro(val c: MacroContext) {
     }
   }
 
-  def flatten(base: Tree, value: Value): List[Tree] = {
+  def flatten(base: Tree, value: Value, publicOnly: Boolean): List[Tree] = {
```

Comment:
Nope 😞 I thought that there will be one but later forgot to remove `publicOnly` parameter.